test/unit/template_importer_test.rb

```diff
@@ -142,5 +142,16 @@ def self.import!(name, text, _metadata)
       assert_equal File.read(get_template('ptable1.erb')), pt.layout
       assert_equal 'Debian', pt.os_family
     end
+
+    test 'purge! removes expected templates' do
+      FactoryGirl.create(:provisioning_template, :name => 'FooBar delete_me')
+      FactoryGirl.create(:provisioning_template, :name => 'FooBar delete_me_too')
```

Comment:
Ah, that it's not just a singleton delete? I suppose thats fair. Up to you then, but add a comment explaning that if you want to leave it in.